securitas/templates/_form_macros.html

```diff
@@ -1,24 +1,39 @@
 {% macro with_errors(field) %}
-  {% if kwargs.pop('nolabel', '') %}
-    {% set label = False %}
-  {% else %}
-    {% set label = True %}
-  {% endif %}
-  {% if field.errors %}
-    {% set css_class = 'form-control is-invalid ' + kwargs.pop('class', '') %}
-  {% else %}
-    {% set css_class = 'form-control ' + kwargs.pop('class', '') %}
-  {% endif %}
+  {% set css_class = 'form-control ' + kwargs.pop('class', '') %}
 
-  {% if label %}
-    <label for="{{ field.name }}" class="mb-0 font-weight-bold">{{ field.label }}</label>
-  {% endif %}
-  {{ field(class=css_class, **kwargs) }}
-  
-  {% if field.errors %}
-  <div class="invalid-feedback">
-      {{ field.errors|map('escape')|join(' ') }}
-  </div>
+  {% if field.type == 'FieldList' %}
+    {# append 4 empty entries at the bottom of the fieldlist #}
+    {% for i in range(4) %}
```

Comment:
4 additional entries seem like a bit too many, no? I would go with 2 (and remember, 640k should be enough for anyone ;-) ).

Reply:
agreed. 2 it is